src/sentry/monitors/tasks.py

```diff
@@ -38,8 +38,8 @@ def check_monitors(current_datetime=None):
     if current_datetime is None:
         current_datetime = timezone.now()
 
-    qs = Monitor.objects.filter(
-        type__in=[MonitorType.CRON_JOB], next_checkin__lt=current_datetime
+    qs = MonitorEnvironment.objects.filter(
+        monitor__type__in=[MonitorType.CRON_JOB], next_checkin__lt=current_datetime
     ).exclude(
         status__in=[
```

Comment:
I think we need to exclude monitors that are disabled, we don't disable monitor environments yet i think right?

```suggestion
        monitor__status__in=[
```

Reply:
ah right. will just chain it instead of `Q`